Source/loadsave.cpp

```diff
@@ -1677,20 +1681,43 @@ bool IsHeaderValid(uint32_t magicNumber)
 	return false;
 }
 
+// Returns the size of the hotkeys file with the number of hotkeys passed and if a header with the number of hotkeys is present in the file
+size_t HotkeysSize(size_t nHotkeys=NUM_HOTKEYS, bool header=1) {
+	return (sizeof(uint8_t)*header) + (nHotkeys * sizeof(int32_t)) + (nHotkeys * sizeof(int8_t) + sizeof(int32_t) + sizeof(int8_t));

```

Comment:
Integer promotion allows implicit conversion from `bool` to `int` but this isn't great for readability/maintainability. devilutionX uses both [modernize-use-bool-literals](https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-bool-literals.html) and [readability-implicit-bool-conversion](https://clang.llvm.org/extra/clang-tidy/checks/readability-implicit-bool-conversion.html) which this use of `header` violates.

Reply:
Ok, noted, removed it as per your previous suggestion.